.github/workflows/wix.yml

```diff
@@ -54,13 +52,30 @@ jobs:
     steps:
       - uses: actions/checkout@9bb56186c3b09b4f86b1c65136769dd318469633 # v4.1.2
 
-      - name: Install dependencies
+      - name: Download IcedTea-Web
         run: |
-          choco install --no-progress wget
+          Invoke-WebRequest -Uri "https://github.com/AdoptOpenJDK/IcedTea-Web/releases/download/${{ matrix.ICEDTEA_WEB_VERSION }}/${{ matrix.ICEDTEA_WEB_VERSION }}.win.bin.zip" `
```

Comment:
I think the default further above should be upgraded to 1.8.8 (same with the Java version up above, move to latest).

Reply:
Yeah I didn't want to over complicate this PR, I'll bump it as part of Josh's larger Wix upgrade